assets/behaviors/advancedTasksForOreon.behavior

```diff
@@ -0,0 +1,16 @@
+{
+    selector : [
+        needs_food,
+        needs_sleep,
+        {
+            timeout : {
+                time : 200,
+                child : {
+                    log : {
+                        message : "Sleeping zzzzz"
+                    }
+                }
+            }
+        }
+    ]
+}
```

Comment:
What I'm not sure IntelliJ will catch: best practice of leaving a newline at the bottom of every text file. GitHub catches it tho